rocketpy/Rocket.py

```diff
@@ -652,8 +684,20 @@ def cnalfa1(cn):
             # Takes an approximation to an angular coefficient
             clalpha = cldata.differentiate(x=0, dx=1e-2)
 
+            # Parameters for Roll Moment
+            if delta != 0:
+                trapezoidalConstant = ((Yr) / 2) * (radius ** 2) * s
+                trapezoidalConstant += ((Cr + 2 * Ct) / 3) * radius * (s ** 2)
+                trapezoidalConstant += ((Cr + 3 * Ct) / 12) * (s ** 3)
+
+                clfDelta = n * (Ymac + radius) * clalpha / d
+
+                cldOmega = n * clalpha * np.cos(delta) * trapezoidalConstant / (Af * d)
+
+                rollParameters = [clfDelta, cldOmega, delta]
```

Comment:
This is a repeated code, which can be hard to maintain (we may change its first appearance above and forget to change this one).

Is there any way for us to remove this part of the code from the "airfoil if statement" and thus make it appear only once? Perhaps we can treat all fins as airfoils and have a constant 2pi as the cl_alpha for the flat plate airfoil (which would mean setting a default 2*np.pi as the airfoil entry, something like that, or even change the name of the "airfoil" entry to "cl_alpha_curve")

Anyways, just suggesting ideas to improve code readability and mantainability.

Reply:
I partially fixed this but this comment actually got me looking into the airfoil calculations again. I should make a PR soon changing this part of the code